File: python/입문/한_번만_등장한_문자.py
# 한 번만 등장한 문자 문제

def solution(s):
    answer = {}
    for str in s:
        if str in answer:
            answer[str] += 1
        else:
            answer[str] = 1
    return "".join(sorted([key for key, value in answer.items() if value == 1]))

# 딕셔너리에 키가 있는지 확인하는 방법

File: java/입문/한_번만_등장한_문자.java
import java.util.Arrays;

class Solution {
    public String solution(String s) {
        String answer = "";
        for(int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            if(s.length() - s.replace(""+c,"").length() == 1) {
                answer += c + "";
            }
        }
        
        char[] chars = answer.toCharArray();
        Arrays.sort(chars);
        
        return String.valueOf(chars);
    }
}